rosetta/scripts/get_balance.go

```diff
@@ -12,19 +12,21 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-package retriever
+package scripts
 
-const getBalance = `
-// This script reads the balance field of an account's FlowToken Balance
+// Adopted from:
+// https://github.com/onflow/flow-core-contracts/blob/master/transactions/flowToken/scripts/get_balance.cdc
 
-import FungibleToken from 0x9a0766d93b6608b7
-import FlowToken from 0x7e60df042a9c0868
+const getBalance = `// This script reads the balance field of an account's FlowToken Balance
+
+import FungibleToken from 0x{{.Params.FungibleToken}}
+import FlowToken from 0x{{.Token.Address}}
```

Comment:
Perhaps a dumb question - if we're replacing the `.Token.Type` below in the template, meaning that the tokens can have different types.. will we always import `FlowToken` here, or will we also need different imports for other tokens?

e.g. `import {{.Token.Type}} from 0x{{.Token.Address}}` ?

Reply:
That's a good catch. Yes, the type is imported, so it should also be replaced there.